src/ui/components/Icon/styles.scss

```diff
@@ -32,3 +32,29 @@
   background: url('./user-dark.svg') center no-repeat;
   background-size: contain;
 }
+
+.Icon-browse {
+  background: url('./bullet-list.svg') 50% 50% no-repeat;
+  background-size: cover;
+  height: 30px;
+  margin-right: 1em;
+  width: 30px;
+}
+
+.Icon-persona {
+  background: url('./themes.svg') 50% 50% no-repeat;
+  background-size: cover;
+  display: inline-block;
+  height: 50px;
+  margin-bottom: 15px;
```

Comment:
Hahaha yeah when I was working on Mozilla's Network Pulse, my mentor mmavis shared the same link. It is a very nicely written blog.